### PR TITLE
net/netmgr: Add initlist, releaselist ops

### PR DESCRIPTION
**Pull request by junmin-kim**
netstack_ops has initlist, releaselist operation

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [d08ca6c7c5bb1b66300ca7c6e11b2f75bf609c5a] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [d08ca6c7c5bb1b66300ca7c6e11b2f75bf609c5a] - Code Rule Check OK.

File: os/net/netmgr/net_vfs.c [hunk 1]
```diff
@@ -421,6 +421,6 @@ void net_initlist(FAR struct socketlist *list)
 
 void net_releaselist(FAR struct socketlist *list)
 {
-	/*	Todo */
-	return;
+	struct netstack *stk = get_netstack(TR_UDS);
+	stk->ops->releaselist(list);
```
Comment (pillip8282) [inline os/net/netmgr/net_vfs.c]: you'd better to cast list variable.

Comment (junmin-kim) [reply os/net/netmgr/net_vfs.c]: Fixed

**Comment by seinfra** [comment]
Target : [2d1b75707ecbb835605b07b356ab42955f0b2abd] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [2d1b75707ecbb835605b07b356ab42955f0b2abd] - Code Rule Check OK.

**Pull request merged**